server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
+
+WORKDIR /app 
+
+COPY package.json /app/package.json
+COPY package-lock.json /app/package-lock.json
+# COPY . /app
+
+RUN npm ci 
+COPY . /app 
+
+EXPOSE 3000 
```

Comment:
App is running on port 4000. With this we may not required to pass the `expose` parameter from the CLI